.atomist/tests/project/AddLicenseTest.feature

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
The comment character in Gherkin feature files in `#`, there is no multiline comment.